x-pack/plugins/ml/public/data_frame/common/aggregations.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export type AggName = string;
+export type FieldName = string;
+
+export function isAggName(arg: any): arg is AggName {
```

Comment:
Would `isValidAggName` be a clearer name for this function?

Reply:
I named this after the TypeScript docs for type guards [here](https://www.typescriptlang.org/docs/handbook/advanced-types.html) (see section "User-Defined Type Guards"). It's also the same for `isAnnotation()` for annotations. So I'd suggest to keep as is.

Reply:
OK. Fair enough!